schema/project-level/codelists/documentType.csv

```diff
@@ -64,6 +64,26 @@ contractAmendment,Contract amendment,A legal document that changes or adds to th
 designReport,Project design report,"A record of the justification for the design approach adopted in the project, typically issued before the works start, by the consultant or firm that will supervise the works.",OC4IDS
 paymentCertificate,Payment certificate,A document certifying that the work specified on a supplier's estimate or invoice is complete and that payment to the supplier is approved.,OC4IDS
 escalationApproval,Escalation approval,Documentation of one or more approved escalations of the contract price during the project life-cycle.,OC4IDS
+procurementStrategyRiskAssessment,Procurement strategy risk assessment,"Documentation of the strategic risk assessment carried out for this contracting process addressing in-house capabilities, contract packaging decisions and risk allocation decisions.",OC4IDS
+lifeCycleCostMethodology,Life cycle cost methodology,Documentation of the methodology for calculating the life cycle cost of the assets.,OC4IDS
+costBenefitAnalysis,Cost-benefit analysis,Documentation of the economic cost-benefit analysis carried out for the project.,OC4IDS
+environmentalExemption,Environmental exemption,Documentation of any environmental exemptions or amnesties granted to the project by a relevant authority.,OC4IDS
+climateAndDisasterRiskAssessment,Climate and disaster risk assessment,Documentation of assessments of the project and resulting assets' risk in relation to climate change and natural disasters.,OC4IDS
```

Comment:
Or, just remove `and resulting assets'`, since that's implied by pretty much every other code.

```suggestion
climateAndDisasterRiskAssessment,Climate and disaster risk assessment,Documentation of assessments of the project's and resulting assets' risk in relation to climate change and natural disasters.,OC4IDS
```